notify/email/email.go

```diff
@@ -76,8 +77,25 @@ func (c *NotifyConfig) SendMail(subject string, message string) error {
 	}
 	body += "\r\n" + message
 
+	// support "," and ";"
+	split := func(r rune) bool {
+		return r == ';' || r == ','
+	}
+
+	recipients := strings.FieldsFunc(c.To, split)
+
 	auth := smtp.PlainAuth("", c.User, c.Pass, host)
 
+	if !c.SSL {
+		msg := []byte(body)
+		err = smtp.SendMail(c.Server, auth, c.User, recipients, msg)
+		if err != nil {
+			return err
+		}
+
+		return nil
+	}
```

Comment:
Can we just use `client, err := smtp.Dial(c.Server)` when ssl is false, so the code logic can be keep same?

Reply:
That's what I did at first, but many details need to be handled, and encountered some mistakes.
I found The function SendMail provided by smtp implements this logic, so we don't need to repeat it.

Reply:
so, do you mean the failure of  `smtp.SendEmail()` would impact the `smtp.NewClient()` and the rest SMTP operations?

Reply:
I mean, if I use `client, err := smtp.Dial(c.Server)` when ssl is false, I need to deal with some details.
For example, I need to use `err = c.Hello()` etc.
If I ignore these, it will report an error.
This is inconvenient. I think it is a good choice to use SendMail provided by smtp directly.